openmc/deplete/abc.py

```diff
@@ -835,6 +847,12 @@ def integrate(self, final_step=True, output=True):
 
         self.operator.finalize()
 
+    def set_transfer_rate(self, material, elements, transfer_rate,
+                        transfer_rate_units='1/s', destination_material=None):
+
+        self.transfer_rates = _TransferRates(self.operator, self.operator.model)
```

Comment:
This won't work if there are multiple calls to `set_transfer_rate` since it creates a new object each time.